packages/kit/src/views/Setting/pages/Tab/SettingListSubModal.tsx

```
@@ -0,0 +1,112 @@
+  name,
+  showSearchView = true,
+}: {
+  name: ISettingName;
+  showSearchView?: boolean;
+}) {
+  const settingsConfig = useSettingsConfig();
+  const isTabNavigator = useIsTabNavigator();
+  const configList = useMemo(() => {
+    return settingsConfig
+      .find((item) => item?.title === name)
+      ?.configs.filter((item) => item && item.length);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)